exonum-java-binding/testkit/src/test/java/com.exonum.binding.testkit/TestKitTest.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.testkit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.exonum.binding.service.AbstractServiceModule;
+import com.exonum.binding.service.Node;
+import com.exonum.binding.service.Service;
+import com.exonum.binding.service.ServiceModule;
+import com.exonum.binding.service.TransactionConverter;
+import com.exonum.binding.transaction.RawTransaction;
+import com.exonum.binding.transaction.Transaction;
+import com.google.inject.Singleton;
+import io.vertx.ext.web.Router;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+class TestKitTest {
+
+  @Test
+  void createTestKitForSingleService() {
+    TestKit testKit = TestKit.forService(TestServiceModule.class);
+    Service service = testKit.getService(TestService.SERVICE_ID, TestService.class);
+    assertEquals(service.getId(), TestService.SERVICE_ID);
```

Comment:
Shan't we test the other side-effects of creating a testkit (= service initialization, mounting of web handlers)?
I think we can do that with a service implementation that produces some effects in its #initialize and returns
some value.

Reply:
I was planning to add that later, with the native part, rather than writing and ignoring them now. Should I do the latter?

Reply:
I think either is OK, if we don't forget to add the proper tests/update the already added to test all aspects.

Reply:
Ok, I'll create a separate task for that then.